core/src/err.rs

```diff
@@ -95,19 +95,11 @@ impl Error {
             | Error::InvalidClientOptions(m)
             | Error::QueryCursorUpdate(m)
             | Error::QueryExecutionFailed(m)
-            | Error::MongoParseConnectionString(m) => {
-                match m.kind.as_ref() {
-                    ErrorKind::Command(command_error) => command_error.code,
-                    // errors other than command errors probably will not concern us, but
-                    // the following is included for completeness.
-                    ErrorKind::BulkWrite(BulkWriteFailure {
```

Comment:
Why did you remove BulkWrite and not Write?

Reply:
BulkWrite went away in rust driver 3.0

Reply:
Hm, the compiler told me it didn't exist. I'm going to put it back in